File: 27-remove-element/remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
      int[] ar = new int[nums.length];
      int j=0;
      for(int i=0;i<nums.length;i++)
      {
          if(nums[i]!=val)
          {
              ar[j++] = nums[i];
          }
      }
      for(int i=0;i<j;i++)
      {
          nums[i] = ar[i];
      }
      return j;
    }
}

File: 27-remove-element/remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        vector<int> res;
        for(int i: nums)
        {
            if(i!=val)
            {
                res.push_back(i);
            }
        }
        nums.swap(res);
        return nums.size();
    }
};